ReactiveCocoa/Objective-C/UITextView+RACSignalSupport.m

```diff
@@ -35,12 +35,7 @@ - (RACDelegateProxy *)rac_delegateProxy {
 }
 
 - (RACSignal *)rac_textSignal {
-	@weakify(self);
-	RACSignal *signal = [[[[[RACSignal
-		defer:^{
-			@strongify(self);
-			return [RACSignal return:RACTuplePack(self)];
-		}]
```

Comment:
I think this was done to make the returned `RACSignal` not retain `self`. I think this should be reverted.